taxonomies/ingredients.txt

```diff
@@ -1480,7 +1482,7 @@ wikidata:en:Q191907
 wikipedia:en:https://en.wikipedia.org/wiki/Carotenoid
 # ingredient/fr:caroténoïde has 2195 products in 12 languages @2019-04-29
 
-
+<en:paprika
```

Comment:
paprika extract is not really paprika, we don't want a parent for it. It's the same for things like orange flavouring, we don't want orange as a parent.

Reply:
Just for paprika or all extracts?